consensus/dummy/dynamic_fee_window.go

```diff
@@ -0,0 +1,92 @@
+// (c) 2019-2025, Ava Labs, Inc. All rights reserved.
```

Comment:
nit shouldn't this be:

```suggestion
// (c) 2025, Ava Labs, Inc. All rights reserved.
```

since it's a new file